ree/reeCorrectedFlux.cc

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include <apfCavityOp.h>
+#include "apfElement.h"
+#include "crv.h"
```

Comment:
@Samiullah-Malik 

I don't think you need anything from from the `crv` here. If that's true, remove the header files that start with `crv`.

Reply:
The function crv::isBoundaryEntity() is used here, so I have kept crv here.

Reply:
@Samiullah-Malik 

please remove both crv.h and crvShape.h and use the following condition wherever you use "isBoundaryEntity"

```
m->getModelType(m->toModel(e)) < m->getDimension()
```

I think letting this part of the code to depend on `crv` libraries can cause problems in the future.